nimble/host/mesh/syscfg.yml

```diff
@@ -524,6 +524,11 @@ syscfg.defs:
             Secondary sysinit stage for BLE mesh functionality.
         value: 1000
 
+    BLE_MESH_SEG_RETRANSMIT_ATTEMPTS:
+        description: >
+            Number of retransmit attempts (after the initial transmit) per segment.
```

Comment:
in such case we should better add restriction in package for this setting

Reply:
@rymanluk I was going through the spec to confirm the minimum allowed value by the spec. I couldn't found anything specific about 'minimum number of segment re-transmission'. Although, the spec states that the TTL value for relaying a message should be minimum 2. 

Also I have tried setting BLE_MESH_SEG_RETRANSMIT_ATTEMPTS to 0 and it worked without any error. 

And yes you are right, re-transmissions happens only when segments are not acked. As I am publishing over a group address and my message is of 3 segments if I set BLE_MESH_SEG_RETRANSMIT_ATTEMPTS to 4, the time difference between receiving a message is around 10 to 15 secs but if I set the value to 0 it just ~3.5 sec. So I preferred sending a message multiple times rather than sending each segment multiple times. Is this an incorrect approach?

Reply:
hmm or maybe minimum should be 1 retransmission. I got confused by reading this from the Mesh Profile spec
**3.5.3.3 Segmentation behavior**
"Each Lower Transport PDU for an Upper Transport PDU shall be transmitted at least two times unless acknowledged earlier."

Reply:
Well, according to 3.5.3.3, you are right. Shall I go ahead and submit PR with restriction for minimum re-transmission to 2?

Reply:
Yes please do. Please also add missing MYNEWT_VALs in porting examples. Thanks